src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -709,7 +709,7 @@ export default class ConvasController {
      * @param aircraft
      */
     canvas_draw_separation_indicator(cc, aircraft) {
-        if (aircraft.category === FLIGHT_CATEGORY.DEPARTURE) {
+        if (aircraft.category === FLIGHT_CATEGORY.DEPARTURE || !GameController.shouldUseTrailingSeparator(aircraft)) {
```

Comment:
yeah, `AircraftModel.isDeparture()` and `AircraftModel.isArrival()`. I'll just make that change and merge then. :shipit: